app/src/main/java/io/fabric/samples/cannonball/activity/PoemPopularActivity.java

```diff
@@ -61,37 +60,19 @@ public void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_tweet_list);
 
         setUpViews();
-
-        loadTweets();
     }
 
     private void setUpViews() {
-        setUpBack();
         setUpPopularList();
+        setUpBack();
     }
 
     private void setUpPopularList() {
-        adapter = new TweetViewAdapter(PoemPopularActivity.this);
-        setListAdapter(adapter);
+        SearchTimeline searchTimeline = new SearchTimeline.Builder().query(SEARCH_QUERY).build();
 
+        final TweetTimelineListAdapter timelineAdapter = new TweetTimelineListAdapter(this, searchTimeline);
```

Comment:
If you want it (you may not), the TweetTimelineListAdapter now (as of Twitter 1.5) has a refresh method which takes a callback so you could show the same toast as before on errors.